README.md

```diff
@@ -1,150 +1,119 @@
 # workshop-template
 
 This repository is [Software Carpentry](http://software-carpentry.org)'s
-template for creating websites for workshops.  Do *not* fork this
-repository directly on GitHub.  Instead, follow the instructions below
-to create a website repository for your workshop (and possibly a
-second repository for your learners to use in your Git lessons).
-
-> If you are teaching Git, you should create a separate repository for
-> learners to use in that lesson.  You should not have them use the
-> workshop website repository because:
+template for creating websites for workshops.
+Do *not* fork this repository directly on GitHub.
+Instead, please follow the instructions below
+to create a website repository for your workshop.
+Please also read
+[the notes on customizing your website](CUSTOMIZATION.md) and the [FAQ](FAQ.md).
+If you're interested in knowing more about why we do things the way we do,
+please check out the [design notes](DESIGN.md).
+
+> If you are teaching Git,
+> you should create a separate repository for learners to use in that lesson.
+> You should not have them use the workshop website repository because:
 > 
-> *   your workshop website repository contains many files that
->     most learners don't need to see during the lesson, and
+> *   your workshop website repository contains many files
+>     that most learners don't need to see during the lesson,
+>     and
 > 
-> *   you probably don't want to accidentally merge a damaging pull
->     request from a novice Git user into your workshop's website while
->     you are using it to teach.
+> *   you probably don't want to accidentally merge
+>     a damaging pull request from a novice Git user
+>     into your workshop's website while you are using it to teach.
 
-## Semi-Automated Setup
+Once you are done,
+please **send your repository's URL to the Software Carpentry administrator**
+so that it can be included in the main web site.
 
-1.  Download the workshop website creation script from
-    [http://files.software-carpentry.org/swc-workshop-create](http://files.software-carpentry.org/swc-workshop-create).
+## Creating a Repository
 
-2.  Make sure that you are *not* inside another Git repository.
+1.  Go to [http://import.github.com](http://import.github.com).
 
-3.  Run the workshop website creation script with no parameters - it
-    will print a help message telling you what parameters it needs.
+2.  Enter the URL for this template repository, which is
+    `https://github.com/swcarpentry/workshop-template`.
```

Comment:
On Wed, Jan 21, 2015 at 04:01:49AM -0800, Carol Willing wrote:
> >  
> > -3.  Run the workshop website creation script with no parameters - it
> > -    will print a help message telling you what parameters it needs.
> > +2.  Enter the URL for this template repository, which is
> > +    `https://github.com/swcarpentry/workshop-template`.
> 
> I received a 500 error using 'https' in URL. It seems to work with 'http'. 

That's very surprising to me, since I don't think GitHub serves
anything besides 301s from http://github.com/…:

  $ wget -S http://github.com/swcarpentry/workshop-template
  --2015-01-21 09:26:45--  http://github.com/swcarpentry/workshop-template
  Resolving github.com... 192.30.252.130
  Connecting to github.com|192.30.252.130|:80... connected.
  HTTP request sent, awaiting response...
    HTTP/1.1 301 Moved Permanently
    Content-length: 0
    Location: https://github.com/swcarpentry/workshop-template
  …

Maybe try with https://github.com/swcarpentry/workshop-template.git?